### ISSUE TITLE
Canu version and platform hardwired in replong.sh

### ISSUE THREAD

**Issue by rsharris** [ruiguo-bio/replong#3]
Lin 185 of replong.sh is
`canuPath="canu-1.4/Linux-amd64/bin"`

If that particular version of canu and is required, this ought to be listed along with the other  installation requirements in the readme.  Ditto for OS and architecture -- if they are required they ought to be listed (instead of expecting the user to dig through the bash script when things fail).

I doubt they are required.  In which case you could give the user a clue by mentioning that she's expected to hack the shell script to get things to work.

**Comment by rsharris** [comment]
And your install.sh script doesn't install canu in a "canu-1.4" directory.  
git clone https://github.com/ruiguo-bio/canu" creates a directory named "canu", not "canu1.4".

**Comment by bzvew** [comment]
Hi, thank you for your report! I fix that bug according to your second comment. If that bug does not exist, replong should work without pointing out the canu version. Now I make it clear in README.

**Comment by rsharris** [comment]
Thanks.

BTW, the updated replong.sh still presumes the user is running on a machine with linux and amd64 architecture (line 196).  If she's on some other type of machine the canu build won't create the path replong.sh requires.

**Comment by bzvew** [comment]
Yes, RepLong is tested on Ubuntu and Centos. I add those in README, and I cannot insure it will work on osx, because it uses gnu awk and split, which can have different behavior on mac.

**Comment by rsharris** [comment]
I can attest to RepLong not working on a mac, at least not on mine, as that was where I ran my initial attempts.  The problem I ran into was that canu's meryl step seems to need at least 16G of memory even if you tell it you don't have less than that.

It's possible that was related to my problem size.  At that point I was still trying the 165M drosophila genome.

As you note, there may be other things that would have failed on the mac even if I had gotten past that canu hurdle.

**Comment by bzvew** [comment]
In canu/src/pipelines/canu/Configure.pm, you can change some limit of memory. I think a linux server is the right place to run replong.
